src/Theme_Command.php

```diff
@@ -899,4 +915,30 @@ private static function get_template_path( $template ) {
 
 		return $template_path;
 	}
+
+	/**
+	 * Check if current version of the theme is higher than the one available at WP.org
+	 *
+	 * @param string $slug Theme slug.
+	 * @param string $version Theme current version.
+	 *
+	 * @return bool|string
+	 */
+	protected function is_theme_version_valid( $slug, $version ) {
+		// Get Theme Info.
+		$theme_info = themes_api( 'theme_information', array( 'slug' => $slug ) );
+
+		// Return empty string for themes not on WP.org
+		if ( is_wp_error( $theme_info ) ) {
+			return '';
+		}
+
+		// Compare theme version info.
+		if ( version_compare( $version, $theme_info->version, '>' ) ) {
```

Comment:
`version_compare()` with an operator already returns a boolean value, so you can just return the negation of it:
```php
return ! version_compare( $version, $theme_info->version, '>' );
```